awx/api/views/__init__.py

```diff
@@ -4205,7 +4205,8 @@ def retrieve(self, request, *args, **kwargs):
                 # Remove any ANSI escape sequences containing job event data.
                 content = re.sub(r'\x1b\[K(?:[A-Za-z0-9+/=]+\x1b\[\d+D)+\x1b\[K', '', content)
 
-                body = ansiconv.to_html(html.escape(content))
+                conv = Ansi2HTMLConverter()
+                body = conv.convert(html.escape(content))
```

Comment:
If this API is new, make sure to put in a lower boundary in the `.in` file, accompanied by a code comment.

Reply:
Do you mean setting a lowest version for the library?

Reply:
`ansi2html >= VERSION`. For some reason I thought that maybe some older versions might have different API so it'd make sense to advertise the lowest needed. Perhaps, I'm wrong and any version works 🤷‍♂️